Formula/git-remote-gcrypt.rb

```diff
@@ -0,0 +1,21 @@
+class GitRemoteGcrypt < Formula
+  desc "PGP-encrypted git remotes"
+  homepage "https://spwhitton.name/tech/code/git-remote-gcrypt/"
+  url "https://github.com/spwhitton/git-remote-gcrypt/archive/1.0.2.tar.gz"
+  sha256 "002994d60d4b3c9a93452d2655bdf3e761c581159431f743d0827127d88f7be2"
+
+  depends_on "docutils" => :build
+  depends_on :gpg => :run
+
+  def install
+    inreplace "./install.sh", "rst2man", "rst2man.py"
```

Comment:
Why the need for this?

Reply:
`rst2man` is used for generating manpages, and `docutils` provide it as `rst2man.py`.

Reply:
Is this something upstream need telling about? Or is it `rst2man.py` on macOS but `rst2man` on other platforms or? Not sure if you're saying this is an upstream "bug" essentially, or not.

Reply:
In FreeBSD (at least) the [port](//svnweb.freebsd.org/ports/head/textproc/py-docutils/Makefile?view=markup) creates symlinks to the `.py` files:

    SCRIPTS=        rst2html rst2latex rst2man rst2odt rst2odt_prepstyles \
                    rst2pseudoxml rst2s5 rst2xetex rst2xml rstpep2html
    
    post-install:
    .for script in ${SCRIPTS}
            ${LN} -s ${script}.py ${STAGEDIR}${PREFIX}/bin/${script}
    .endfor

I think similar thing happens on other platforms too, as that's how the upstream of `git-remote-gcrypt` has implemented it.

Reply:
Hmm. If it is a widespread thing perhaps that's something Homebrew should consider picking up. Will punt to @ilovezfs for thoughts 🙃.

Reply:
Sounds like an upstream (git-remote-gcrypt) bug to me. There's nothing I can see in docutils itself that would produce such a thing so people shouldn't assume it's going to exist.

Reply:
Maybe, but it seems like many distributions are just prefering to strip `.py` extension from scripts. Following is from a Debian Stretch host:

    λ cat /var/lib/dpkg/info/python-docutils.postinst
    #!/bin/sh
    set -e
    
    if [ "$1" = configure ]
    then
            for exe in /usr/share/docutils/scripts/python2/*
            do
                    update-alternatives --install /usr/bin/${exe##*/} ${exe##*/} $exe 30
            done
    fi
    
    
    # Automatically added by dh_python2:
    if which pycompile >/dev/null 2>&1; then
            pycompile -p python-docutils 
    fi
    
    # End automatically added section
    
    λ dpkg -L python-docutils |fgrep /scripts/python2/
    /usr/share/docutils/scripts/python2/rst-buildhtml
    /usr/share/docutils/scripts/python2/rst2html
    /usr/share/docutils/scripts/python2/rst2html5
    /usr/share/docutils/scripts/python2/rst2latex
    /usr/share/docutils/scripts/python2/rst2man
    /usr/share/docutils/scripts/python2/rst2odt
    /usr/share/docutils/scripts/python2/rst2odt_prepstyles
    /usr/share/docutils/scripts/python2/rst2pseudoxml
    /usr/share/docutils/scripts/python2/rst2s5
    /usr/share/docutils/scripts/python2/rst2xetex
    /usr/share/docutils/scripts/python2/rst2xml
    /usr/share/docutils/scripts/python2/rstpep2html
    /usr/share/docutils/scripts/python2/rst2html4

And following is from a Fedora Core 25 RPM archive:

    λ rpm -qlp python-docutils-0.12-0.7.20140510svn7747.fc25.noarch.rpm |fgrep /bin/
    error: cannot open Packages database in /var/lib/rpm
    warning: python-docutils-0.12-0.7.20140510svn7747.fc25.noarch.rpm: Header V3 RSA/SHA256 Signature, key ID fdb19c98: NOKEY
    /usr/bin/rst2html
    /usr/bin/rst2latex
    /usr/bin/rst2man
    /usr/bin/rst2odt
    /usr/bin/rst2odt_prepstyles
    /usr/bin/rst2pseudoxml
    /usr/bin/rst2s5
    /usr/bin/rst2xetex
    /usr/bin/rst2xml
    /usr/bin/rstpep2html